data/deep missions.txt

```diff
@@ -40,24 +40,29 @@ mission "Deep: Syndicate Convoy"
 	to offer
 		"mystery cube" >= 10
 		or
-			random < 30
+			random < 20
 			and
 				"deep convoy" == 1
-				random < 40
+				random < 30
 			and
 				"deep convoy" == 2
-				random < 50
+				random < 40
 		
 	on offer
 		conversation
-			`You are approached by a man in a Deep Security uniform who asks you if you would be willing to escort a few freighters to <stopovers> and back by <date>. "We would normally have DS ships do it," he says, "but we're short on available ships given certain circumstances."`
+			`You are approached by a man in a Deep Security uniform who asks you if you would be willing to escort a few freighters to <stopovers> and back by <date>. "We would normally have our own fleet do it," he says, "but we're short on available ships given certain circumstances."`
 			choice
 				`	"Sure, I can do that."`
 				`	"Sorry, but I will not be traveling in that direction any time soon."`
-					decline
+					defer
```

Comment:
I don't want these missions to constantly pop up until the player finally agrees to do them. This is also why you only need to complete 2 of the 3 missions in order to continue down the line. One can be declined should the player not want to do it, but if they continually decline to help the Deep then they won't get the Bactrian.